From db39541d6257bdb07302896c9465383b36f54772 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Dec 2024 20:20:03 +0800
Subject: [PATCH 160/166] AOSCOS: MIPS: loongson64_cpuautoplug: rename
 get_idle_time() to gs464_get_idle_time()

The original naminig for this function was too generic and conflicted with
another definition in include/linux/kernel_stat.h.

Rename to gs464_get_idle_time() as a workaround.

Fixes: "AOSCOS: MIPS: Loongson 3: Add CPU Autoplug support"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 arch/mips/loongson64/loongson64_cpuautoplug.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/mips/loongson64/loongson64_cpuautoplug.c b/arch/mips/loongson64/loongson64_cpuautoplug.c
index 37a930970345..a69a9ef9b11e 100644
--- a/arch/mips/loongson64/loongson64_cpuautoplug.c
+++ b/arch/mips/loongson64/loongson64_cpuautoplug.c
@@ -242,7 +242,7 @@ static inline u64 get_idle_time_jiffy(u64 *wall)
 	return div_u64(idle_time, NSEC_PER_USEC);
 }
 
-static inline u64 get_idle_time(u64 *wall)
+static inline u64 gs464_get_idle_time(u64 *wall)
 {
 	unsigned int cpu;
 	u64 idle_time = 0;
@@ -316,7 +316,7 @@ static void do_autoplug_timer(struct work_struct *work)
 	}
 
 	/* based on cpu load */
-	cur_idle_time = get_idle_time(&cur_wall_time);
+	cur_idle_time = gs464_get_idle_time(&cur_wall_time);
 	if (cur_wall_time == 0) {
 		cur_wall_time = jiffies64_to_nsecs(get_jiffies_64());
 		cur_wall_time = div_u64(cur_wall_time, NSEC_PER_USEC);
-- 
2.47.1

